rx-java2-gen/src/main/java/io/vertx/reactivex/RxHelper.java

```diff
@@ -65,4 +72,19 @@ public static Scheduler blockingScheduler(WorkerExecutor executor) {
     return new ContextScheduler(executor, false);
   }
 
+  public static <T> Subscriber<T> toSubscriber(WriteStream<T> stream, Consumer<Throwable> onError, Runnable onComplete) {
```

Comment:
javadoc ?

Reply:
Yeah, in the email to `vertx-dev` I said docs were planned but I wanted to share the code early.